tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js

```diff
@@ -21,6 +21,8 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
   describe("Given the user has added some members to the household and is on the Hub", () => {
     before("Open survey and add household members", () => {
       browser.openQuestionnaire("test_repeating_sections_with_hub_and_spoke.json");
+      // Accept cookies, since they push submit button outside window
```

Comment:
Sorry, I don't quite understand this comment and how this relates to this story

Reply:
For some reason the submit button pushed the window down and it couldn't submit so we had to accept cookie banner to bring that submit button within window. it is weird only this test was failing because of that

Reply:
It's the EQ_RUN_FUNCTIONAL_TESTS_HEADLESS issue again, Doing this to one test isn't the best. I would prefer we updated wdio.config window-size if it works, although still not ideal, because in time  the reason for doing this is lost. Even a comment saying this has been done down to a headless window-size issue would be better

Reply:
If the comment is staying, it should mention the banner, otherwise it is a bid confusing as to why cookies has anything to do with it. The significance being the banner pushes the button outside the windows.

Reply:
updated comment now , it is headless window size issue again. I tried with few window size combination. it works on local but fails in actions